app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+  <div class = "col-xs-5" >
+    <%= form_for [@product, @product.reviews.build], class: "form-group" do |f| %>
+      <h3>Rating:</h3>
+      <%= f.label :rating, render_stars(1), class: "radio-inline star" %>
```

Comment:
I think you could probably do this in a loop rather than re-writing the label & radio button markup for each review value